discord/utils.py

```diff
@@ -141,13 +141,16 @@ def oauth_url(client_id, permissions=None, guild=None, redirect_uri=None):
         The guild to pre-select in the authorization screen, if available.
     redirect_uri: :class:`str`
         An optional valid redirect URI.
+    scopes: List[:class:`str`]
```

Comment:
This could be changed to `Iterator` but I'm not sure if that fits with the rest of the documentation. `set` could also be used

Reply:
It would be Iterable not Iterator

Reply:
Yeah `Iterable` would work here.